src/components/Card.jsx

```diff
@@ -1,26 +1,52 @@
+import { useState } from 'react';
+
 import { calculateDateDifference, formatDate } from '../utils/dateCalculate';
+
 import defaultImg from '../assets/logo.svg';
 import starIcon from '../assets/icon/star.svg';
 import kebabIcon from '../assets/icon/kebab.svg';
 
 import './Card.css'
+import ModalLinkAdd from './modals/ModalLinkAdd';
+
+const Card = ({imgSrc, createdAt, title, description, url, folderName, folderList}) => {
+  const [isOpenKebab, setIsOpenKebab] = useState(false);
+  const [isOpenModalLinkAdd, setIsOpenModalLinkAdd] = useState(false);
+
+  const handleClickKebab = () => {
+    setIsOpenKebab(!isOpenKebab);
+  }
+
+  const handleClickModalLinkAdd = () => {
+    setIsOpenModalLinkAdd(!isOpenModalLinkAdd);
```

Comment:
여기도 동일합니다~
```suggestion
    setIsOpenModalLinkAdd((isOpenModalLinkAdd) => !isOpenModalLinkAdd);
```

Reply:
일반적으로는 이렇게 표현하기도 해서 알아두시면 좋을 것 같아요~
```suggestion
    setIsOpenModalLinkAdd((prev) => !prev);
```